modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -56,6 +63,18 @@
 |`./dynamic-plugins/dist/parfuemerie-douglas-scaffolder-backend-module-azure-repositories`
 
 
+|Backend  |`https://npmjs.com/package/@pagerduty/backstage-plugin-backend/v/0.6.1[@pagerduty/backstage-plugin-backend]` |0.6.1 
```

Comment:
I think the name of plugin is pagerduty, could you please confirm?
```suggestion
|PagerDuty  |`https://npmjs.com/package/@pagerduty/backstage-plugin-backend/v/0.6.1[@pagerduty/backstage-plugin-backend]` |0.6.1 
```

Reply:
Good spot @hmanwani-rh - it looks like a tweak is required to the table generation script.

Reply:
Probably a bug in this section https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/modules/dynamic-plugins/rhdh-supported-plugins.sh#L220-L233

I'll have a look.

Reply:
fix in https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/505